test/Microsoft.IdentityModel.Tokens.Saml.Tests/SamlSerializerTests.cs

```diff
@@ -106,6 +106,13 @@ public static TheoryData<SamlTheoryData> ReadActionTheoryData
                         ActionTestSet = ReferenceSaml.SamlActionValid,
                         SamlSerializer = new SamlSerializerPublic(),
                         TestId = nameof(ReferenceSaml.SamlActionValid)
+                    },
+                    new SamlTheoryData
```

Comment:
Add XML element as property then we can get rid of testsets.

Reply:
I did this for Saml2SerializerTests, but it's somewhat more complicated to do this for SamlSerializerTests because a lot of the test sets rely on information created in other test sets (ex.  a SamlAssertionTestSet relies on Conditions created in a SamlConditionTestSet). Would it make sense to extract the information from the test sets and store them as separate properties in ReferenceSaml.cs?